tools/benchmark.cr

```diff
@@ -0,0 +1,57 @@
+require "../src/pluto"
+
+def benchmark_memory
+  bytes_before_measure = GC.stats.total_bytes
+  yield
+  (GC.stats.total_bytes - bytes_before_measure).to_i64
+end
+
+def benchmark_time
+  time = Time.monotonic
+  yield
+  Time.monotonic - time
+end
+
+def output_results(total : Hash(String, NamedTuple(memory: Int64, time: Int64)))
```

Comment:
Yeah, it felt a bit convoluted. I didn't use `String.build` since all of the output was going straight to standard out, so the intermediary builder wasn't likely going to offer anything. I can experiment though, might come up with something nicer to read.

Reply:
IMO, it is not only faster, but also looks better/easier to read.